plugins/aac-plantuml/aac-plantuml.yaml

```diff
@@ -6,15 +6,12 @@ model:
       type: command
       description: Converts an AaC model to Plant ULM component diagram
       input:
-        - name: archFile
-          type: file
         - name: parsed_model
           type: map
       acceptance:
         - scenario: Output PlantUML component diagram for valid architecture.
           given:
-            - The {{puml.input.archFile}} contains a valid architecture specification.
-            - The {{puml.input.parsed_model}} contains the parsed content from archFile.
+            - The {{puml.input.parsed_model}} contains the parsed content from architecture_file.
```

Comment:
Where is `architecture_file` coming from? Should it still be an input for the `puml-component` behavior?